projects/ngrx.io/content/guide/effects/index.md

```diff
@@ -46,10 +51,12 @@ You also have the corresponding service that handles the fetching of movies.
 
 <code-example header="movies.service.ts">
 @Injectable({
-  providedIn: 'root'
+  providedIn: 'root',
+  standalone: true,
+  imports: [HttpClient],
```

Comment:
`Injectable` doesn't have `standalone`/`import` properties

```suggestion
```

Reply:
done, i also  add return type for getAll and some imports